src/components/App/index.tsx

```diff
@@ -46,10 +49,18 @@ export const App: FC = () => {
           JSON.stringify(whoAmI?.courses[0])
         );
       }
+      if (!localStorage.getItem(CURRENT_LANG)) {
+        localStorage.setItem(CURRENT_LANG, LANGUAGES[0]);
+      }
       const currentCourse = JSON.parse(
```

Comment:
const currentCourse = localStorage.getItem(CURRENT_COURSE);

Reply:
![Screenshot 2021-03-05 at 11 30 14](https://user-images.githubusercontent.com/50995397/110088660-327c9280-7da6-11eb-8c60-41e91187a130.png)

Reply:
If I don'n parse this string I recieve "English", but I need English